CONTRIBUTING.md

```diff
@@ -47,11 +47,11 @@ This repository contains [docker-compose.yml](./docker-compose.yml) for
 (can be also [used with podman](https://fedoramagazine.org/use-docker-compose-with-podman-to-orchestrate-containers-on-fedora)).
 Before you run it, we suggest that you open the file and read all the comments.
 You can also run only certain pieces of packit-service for local development
-(e.g. worker, database or service/httpd).
+(e.g. worker, database or service).
```

Comment:
```suggestion
(e.g. worker, database or http service).
```
Now I am used to know that the service is an http service, but in the beginning this was confusing, service could be any kind of service. I would keep a way to say this is an http service.

Reply:
:+1: though I'd rather use HTTP.

Reply:
makes sense, will change this 👍